app/models/concerns/sample_sheet.rb

```diff
@@ -45,14 +46,11 @@ def all_libraries_tagged
 
   # Sample is Barcoded field - TEMPORARY: Want IsoSeq samples to return false
   def check_library_tags
-    isoseq_tag_sets = %w[IsoSeq_Primers_12_Barcodes_v1]
-    if libraries.any? do |library|
-         isoseq_tag_sets.include?(library.tag.tag_set.name) if library.tag
-       end
-      return false
+    no_isoseq_tag_sets = libraries.none? do |library|
+      ISOSEQ_TAG_SETS.include?(library.tag.tag_set.name) if library.tag
     end
```

Comment:
I actually meant pull this into a separate method, but this is fine, considering its temporary (and still has the advantage of making what the checks are explicit)